libs/ui-watt/tsconfig.spec.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
     "module": "commonjs",
-    "types": ["jest", "node"]
+    "types": ["jest", "node"],
+    "target": "ES2015"
```

Comment:
question: is there an Issue in Material's repo that tracks the "ProxyZone" error? We should definitely aim for bumping the "target" ES version, so maybe we can create an Issue for tracking it.

Reply:
There aren't any active issue on this, but found the solution here: https://github.com/thymikee/jest-preset-angular/issues/1058

The problem seems to be caused by Zone.JS cannot intercept native async/await which are present in ES2017+